File: js/padArray.js
// REMEMBER TO PSEUDOCODE
exports.pad = (array, minSize, value=null) => {
    if (array.length >= minSize){
        // console.log(array)
        return array;
    }else{
        for (let i = array.length; i < minSize; i++ ){
            array.push(value);
            
        }
        return array;
    }
    // console.log(array)
}
// exports.pad([0,1,2,3], 4,)

File: python/pad_array.py
#REMEMBER TO PSEUDOCODE
def pad(array, min_size, value = None):
    if len(array) >= min_size:
        return array
    else: 
        for i in range(len(array), min_size):
            array.append(value)
    return array
print(pad([0,1,2,3], 4,))
print(pad([0,1,2,3], 5,))
print(pad([0,1,2,3], 5, "Working"))
print(pad([0,2,2,3,6], 8, "Forgot"))
